src/main/java/nl/tudelft/ti2206/group9/audio/AudioPlayer.java

```diff
@@ -0,0 +1,118 @@
+package nl.tudelft.ti2206.group9.audio;
+
+import java.io.File;
+import java.net.MalformedURLException;
+
+import javafx.scene.media.AudioClip;
+import javafx.scene.media.MediaException;
+import nl.tudelft.ti2206.group9.level.State;
+import nl.tudelft.ti2206.group9.util.GameObservable;
+import nl.tudelft.ti2206.group9.util.GameObserver.Category;
+import nl.tudelft.ti2206.group9.util.GameObserver.Error;
+
+/**
+ * Creates an AudioPlayer which you can initialize, start and stop.
+ * @author Mitchell
+ */
+@SuppressWarnings("restriction")
+public class AudioPlayer {
+
+	/** The AudioClip of an AudioPlayer. */
+	private static AudioClip audioClip;
```

Comment:
Can I ask why audioClip is static? This would mean creating more than one AudioPlayer is impossible, which is not what we want.

<logic,review